apps/lobe-chat/1.35.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.35.3
+    image: lobehub/lobe-chat:v1.35.6
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
These changes only include minor adjustments to the LobeChat service configuration file in Kubernetes (k8s). There seem to be no major discrepancies, irregularities, or potential issues detected at this point. However, if there is something specific you've noticed from these changes that requires attention or improvement, please let us know so we can address it.